common/config/.golangci-format.yml

```diff
@@ -13,6 +13,7 @@ run:
   deadline: 20m
   build-tags:
   - integ
+  - integfuzz
```

Comment:
Changes in the common directory come from istio/common-files changes, else these would be overridden with the next common-files update.